upgrade/1.2/Stage_2.md

```diff
@@ -3,7 +3,7 @@
 For each storage node in the cluster, start by following the steps:
```

Comment:
I am not sure if there will be a ceph upgrade, but if there is, then we would be moving it to after k8s